.github/workflows/build-release-images.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Save docker images to artifact
         run: |
           mkdir -p docker-images
-          docker save armada | gzip > docker-images/armada.tar.gz
+          docker save armada-server | gzip > docker-images/armada-server.tar.gz 
```

Comment:
How is this related? @richscott or @dejanzele would this break anything?

Reply:
we can rename the gorelease armada-server output to armada if needed. But as that was added already, i made the change here.